jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/util/Select.java

```diff
@@ -52,10 +53,18 @@ public void addOption(final String text, final String value) {
     }
 
     public void addOption(final String text, final String value, final Boolean selected) {
+        addOption(text, null, value, selected);
+    }
+
+    public void addOption(final String text, final String subText, final String value, final Boolean selected) {
```

Comment:
Although it is not used at the moment I would prefer to keep it as is. This is a general UI component that I would expect to be used in different context, so exposing the relevant attributes here.